lib/node/runner/checker_ballot_transaction.go

```diff
@@ -295,6 +305,10 @@ func ValidateOp(st *storage.LevelDBBackend, source *block.BlockAccount, op opera
 			if casted.Linked != "" && (casted.Amount%common.Unit) != 0 {
 				return errors.FrozenAccountCreationWholeUnit // FIXME
 			}
+		} else {
```

Comment:
why it remains?

Reply:
@kfangw Because there is two cases.
1. general account create frozen account.
2. frozen account create frozen account.
So it is for the second case.